angular/src/docs/app/playground/playground.component.ts

```diff
@@ -6,6 +6,8 @@ import { Component } from '@angular/core';
     <div className="row">
       <div className="example-spacing">
         <!-- Insert Example Here -->
+        <example-drawer-default></example-drawer-default>
+        <example-drawer-openfrom></example-drawer-openfrom>
```

Comment:
Revert changes.

Reply:
@pauljeter @bfbiggs @mathsusan would it be worthwhile keeping playground.ts in the gitignore file?

Reply:
I like that idea .. I keep forgetting to undo it.

Reply:
If we .gitignore it, it won't be in the repo at all.